bsip-0018.md

```diff
@@ -88,104 +92,159 @@ This bug will be fixed. See
-   account is willing to obtain
+the settlement fund itself is too low to create a sufficiently collateralized
+short position (in terms of price feed and MCR), investors could volunteer to
+add the required amount of collateral to the fund and take ownership of the
+resulting short position (collateral+debt).
+
+The proposed operation enables potential investors to "bid" additional
+collateral for taking over part of the debt (or all of it). When enough bids
+have been made to cover the full outstanding debt, and all of them are
+sufficiently collateralized (in terms of price feed and MCR), the
+settlement_fund and the bids are turned into call positions. Finally, the
+settlement_price is removed from the asset, which revives it.
+
+If the available bids cover more than the outstanding debt, bids with a higher
+collateral/debt ratio are preferred over those with a lower ratio. The intent is
+to turn the competition among investors into better collateralized calls, which
+is in the interest of the MPA holders.
+
+### `bid_collateral_operation`
+
+The operation has the following payload:
+
+* `fee`(asset_type): The operation requires a fee to be paid
+* `bidder`(account_type): This account pays the additional collateral and
+  will become the owner of the resulting call position, if the bid is accepted
+* `additional_collateral`(asset_type): Collateral paid by the account in order
+  to support the call position
+* `debt_covered`(asset_type): The amount of debt the account is willing to cover
 
 The operation works as follows:
  
- 1. It pays a fee
- 2. It reduces the account's balance by `debt`.
-    The debt is used to reduce the outstanding shares of the globally settled BitAsset.
- 3. It reduces the account's balance by `collateral`.
-    The collateral is used to initially support the accounts' call position.
-    However, technically, only little additional collateral is required (if the
-    valuation of the collateral hasn't change since the global
-    settlement) if the owner accepts a margin call.
- 4. The global settlement flag is removed from the asset.          
- 5. The asset is re-enabled such that price feeds can be produced again.
- 6. After sufficient price feeds, the asset can be borrowed again.
-
-The required checks for the operation are:
-
- * Has the asset globally settled?
- * Are funds in the settlement pool?
- * `debt` > 0 or `collateral` > 0
- * `obtain_settlement_funds` <= `settlement_pool`
- * the account has sufficient balance to cover `additional_collateral`
-
-If the checks are successful, a `call_order_object` belonging to the
-investor will be created or updated as described above. Then, the
-settlement price and `settlement_fund` will be cleared.
-
-The fee for this operation will be paid by the investor/recoverer. The
-fee is equal to the fee of the `call_order_update` operation.
+1. It pays a fee
+2. If account has already placed a bid on the same MPA, the existing bid is
```

Comment:
Perhaps it is not clear what I meant here.
If the account already has a bid, the existing bid is replaced by the new one. This will be implemented as two steps - first cancel the old bid (step 2 in my list), then create the new bid (step 5 in my list).
Step 3 implements your second suggestion - after the old bid has been canceled, a new bid will only be created if debt_covered is > 0, in other words "creating" a bid with debt_covered = 0 cancels an existing bid.